src-gui/src/renderer/components/modal/introduction/slides/Slide03_PrepareSwap.tsx

```diff
@@ -0,0 +1,15 @@
+import { Typography } from '@material-ui/core'
+import SlideTemplate from './SlideTemplate'
+import imagePath from '../../../../../assets/mockConfigureSwap.webp' 
+
+export default function Slide02_ChooseAMaker(props: slideProps) {
+    return (
+        <SlideTemplate title="Prepare Swap" stepLabel="Step 2" {...props} imagePath={imagePath}>
+            <Typography variant="subtitle1">
+                After providing your Monero redeem address and configuring the
+                Bitcoin refund behavior, you can initiate the swap and deposit
```

Comment:
Here i'd change to "After providing your redeem address and (optionally) providing a refund address, you can initiate the swap by deposit your Bitcoin."